scripts/check-public-api.spec.ts

```diff
@@ -22,28 +23,13 @@ describe('check-public-api', () => {
         }
       }
     });
-    expect(indexFiles('root', 'index.ts')).toEqual([
+    expect(indexFiles('root', '**/index.ts')).toEqual([
```

Comment:
can you add one more test regarding the recursive check, which should fail if a file is missing in a subfolder.